worlds/iris_multiple_sitl.world

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" ?>
+<sdf version="1.5">
+  <world name="default">
+    <!-- A global light source -->
+    <include>
+      <uri>model://sun</uri>
+    </include>
+    <!-- A ground plane -->
+    <include>
+      <uri>model://ground_plane</uri>
+    </include>
+    <include>
+      <uri>model://iris</uri>
+      <pose>0 0 0 0 0 0</pose>
+      <name>iris1</name>
```

Comment:
@tfoote Is it feasible to add tags here that are not part of the SDFormat spec, but have them still being parsed into C++?